packages/vue-components/src/Dropdown.vue

```diff
@@ -112,7 +120,7 @@ export default {
     showDropdownMenu() {
       this.show = true;
       $(this.$refs.dropdown).findChildren('ul').each(ul => ul.classList.toggle('show', true));
-    },
+    }
```

Comment:
let's correct these in another PR; we could enable linting for `dropdown.vue` as well. (the unlinted ones are files that haven't changed much from vue-strap)

Reply:
Alright sure will open another PR for this 👍